applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -436,6 +459,53 @@ int le_audio_user_defined_button_press(void)
 	return ret;
 }
 
+static int button_id_5_pressed(void)
+{
+	int ret;
+
+	LOG_INF("Change broadcast gateway");
+
+	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
+	ERR_CHK(ret);
+
+	ret = bis_headset_cleanup(false);
+	if (ret) {
+		LOG_ERR("Error cleaning up");
+		return ret;
+	}
+
+	/* Wrap broadcaster names */
+	if (++active_bt_device_name >= ARRAY_SIZE(broadcast_device_name)) {
+		active_bt_device_name = 0;
+	}
+	LOG_INF("Switching to %s", broadcast_device_name[active_bt_device_name]);
```

Comment:
```suggestion
	LOG_DBG("Switching to %s", broadcast_device_name[active_bt_device_name]);
```
Since we are printing 'Change broadcast gateway' and then 'Broadcast source found' when it is found